libs/domain/picking/src/components/discard-modal/discard-modal.component.ts

```diff
@@ -0,0 +1,49 @@
+import { ButtonType } from '@spryker-oryx/ui/button';
+import { i18n, Size } from '@spryker-oryx/utilities';
+import { html, LitElement, TemplateResult } from 'lit';
+import { property } from 'lit/decorators.js';
+import { discardModalStyles } from './discard-modal.styles';
+
+export class DiscardModalComponent extends LitElement {
+  static styles = discardModalStyles;
+
+  @property({ type: Boolean }) open?: boolean;
+
+  protected override render(): TemplateResult {
+    return html`
+      <oryx-modal
+        ?open=${this.open}
+        enableFooter
+        preventCloseByBackdrop
+        footerButtonFullWidth
+        @oryx.close=${this.close}
```

Comment:
I'm wondering, should this modal close on esc? Are we preventing closing on backdrop, but not on esc keypress? Looks like both ways should be prevented

Reply:
Yup, both ways should be prevented. `preventCloseByEscape` added